pkg/commands/environment.go

```diff
@@ -59,6 +75,52 @@ var environmentEcrCmd = &cobra.Command{
 	PreRun: upgradeIfNotLatest,
 }
 
+var environmentPlanCmd = &cobra.Command{
+	Use:   "plan",
+	Short: `Do a plan for given namespace`,
```

Comment:
Maybe, `Perform a terraform plan and kubectl dry-run for a given namespace`?

Reply:
What's the action if you don't pass a namespace name? That should be the description here.